src/instrumenter/instrument.ts

```diff
@@ -1655,3 +1592,39 @@ export class FunctionInstrumenter {
         return recipe;
     }
 }
+
+/**
+ * Return the constructor of `contract`. If there is no constructor defined,
+ * add an empty public constructor and return it.
+ *
+ * @param contract
+ * @param factory
+ * @returns
```

Comment:
These lines can be removed as do not provide anything useful,

Reply:
Fixed in dcf955c.